data/json/items/migration.json

```diff
@@ -1617,5 +1617,11 @@
     "id": "folding_bthk_stock",
     "type": "MIGRATION",
     "replace": "high_end_folding_stock"
+  },
+  {
+    "id": "sourdough_starter",
+    "type": "MIGRATION",
+    "replace": "jar_glass_sealed",
+    "contents": [ { "id": "sourdough_starter", "count": 1 } ]
```

Comment:
> the game throws an error invalid recipe id "sourdough_young"

Maybe that's because when you migrate `"sourdough_young"` like this, the item id is removed, so a recipe with that id is no longer valid?

Reply:
It seems like I didn't understand how migrations work, I thought they just convert items with certain ID to other items. In case of `sourdough_starter` the game behaved well: instead of sourdough starters I got glass jars with sourdough starters in them. But it didn't work for `sourough_young` and `sourdough_split`.

So, how do I ensure people don't lose their glass jars in existing saves then?

Reply:
> instead of sourdough starters I got glass jars with sourdough starters in them

If you save and reload again what happens? I think because the migration system doesn't know whether an item has already been migrated, it will migrate the contained sourdough starter again to a jar of sourdough starter.

> the game throws an error invalid recipe id "sourdough_young"

I think you can try change the id of the starter/young/split version of the sourdough and migrate the old sourdough ids to glass jars containing items with the new ids.

Reply:
> I think you can try change the id of the starter/young/split version of the sourdough and migrate the old sourdough ids to glass jars containing items with the new ids.

I did that. The IDs now look a bit ugly, but it works, I tested it, please review.